aplib.net-demo/Assets/Scripts/WFC/SpawnAbleItems.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Assets.Scripts.Wfc
+{
+    /// <summary>
+    /// Represents the spawnable items.
+    /// </summary>
+    [CreateAssetMenu(fileName = "SpawnAbleItems", menuName = "ScriptableObjects/SpawnAbleItems", order = 1)]
+    public class SpawnAbleItems : ScriptableObject
+    {
+        /// <summary>
+        /// Represents the spawnable items count.
+        /// </summary>
+        public List<int> SpawnableItemsCount = new();
+
+        /// <summary>
+        /// Represents the spawnable items.
+        /// </summary>
+        public List<GameObject> SpawnableItems = new();
```

Comment:
Since these are actually pairs, you can do this like so:
```suggestion
        [field: SerializeField]
        public List<SpawnableItem> Items { get; set; }

        [Serializable]
        public class SpawnableItem
        {
            public GameObject Item;
            public int Count;
        }
```